packages/connectors/src/ledger.ts

```diff
@@ -0,0 +1,244 @@
+// @ts-ignore
+import { type EthereumProvider } from '@ledgerhq/connect-kit/dist/umd/index.d.ts'
```

Comment:
I wonder if you could skip the `// @ts-ignore` hack by setting inside the `package.json`:

```json
"alias": {
    "@ledgerhq/connect-kit": "@ledgerhq/connect-kit/lib-umd"
}
```

Reply:
Hey @W3stside No need to alias it, using `import type { ...` instead works fine and doesn't throw typescript errors. Thanks for bringing it up though!